python/postprocessing/modules/btv/btagSFProducer.py

```diff
@@ -5,19 +5,29 @@
 from PhysicsTools.NanoAODTools.postprocessing.framework.datamodel import Collection
 from PhysicsTools.NanoAODTools.postprocessing.framework.eventloop import Module
 
-def is_relevant_syst_for_shape_corr(flavor_btv, syst):
+def is_relevant_syst_for_shape_corr(flavor_btv, syst, self):
     """Returns true if a flavor/syst combination is relevant"""
+    era = self.era
     if flavor_btv == 0:
+        if era.find("FastSim"):
```

Comment:
this check does NOT do what you think it does.  string.find() in python returns -1 if the string is not found.  "-1" evaluates as True!!!.  Please replace this with
```
if "FastSim" in era:
    #blah
```
Also replace it the 2 other places this is used in this function